### DISCUSSION TITLE
Microsoft Auth - How to whitelist multiple Microsoft groups to access concourse

### DISCUSSION THREAD

**Discussion by ybindu** [concourse/concourse#8941] [Help & Support]
I'm new to concourse and I'm not able to get concourse (6.3.0) authentication through Microsoft Azure AD. How to add multiple microsfot AD groups to --microsoft-groups paraameter?

**Comment by taylorsilva** [comment]
Based on https://github.com/jessevdk/go-flags I'd guess you declaring that parameter once per group, like:

```
concourse web -microsoft-groups group1 -microsoft-groups group2
```

Haven't tried myself though, just guessing based on the library used by Concourse.

---

FYI: That is a very old and unsupported version. The only LTS version is v6.8.x.
